docs/aws.md

```diff
@@ -509,6 +509,24 @@ spark-sql --conf spark.sql.catalog.my_catalog=org.apache.iceberg.spark.SparkCata
 
 For more details on using S3 Acceleration, please refer to [Configuring fast, secure file transfers using Amazon S3 Transfer Acceleration](https://docs.aws.amazon.com/AmazonS3/latest/userguide/transfer-acceleration.html).
 
+### S3 Dual-stack
+
+[S3 Dual-stack](https://docs.aws.amazon.com/AmazonS3/latest/userguide/dual-stack-endpoints.html) allows a client to access an S3 bucket through a dual-stack endpoint. 
+When clients make a request to a dual-stack endpoint, the bucket URL resolves to an IPv6 or an IPv4 address.
```

Comment:
I think it should be "the bucket URL resolves to an IPv6 address if possible, otherwise fallback to IPv4"?

Reply:
Thank you for your suggestions. I will update the doc.